gensim/summarization/commons.py

```diff
@@ -3,10 +3,46 @@
 #
 # Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
 
+"""This module provides functions of creatinf graph from sequence of values and
+removing of unreachable nodes.
+
+
+Examples
+--------
+
+Create simple graph and add edges. Let's kake a look at nodes.
+
+>>> gg = build_graph(['Felidae', 'Lion', 'Tiger', 'Wolf'])
+>>> gg.add_edge(("Felidae", "Lion"))
+>>> gg.add_edge(("Felidae", "Tiger"))
+>>> gg.nodes()
+['Felidae', 'Lion', 'Tiger', 'Wolf']
+
+Remove nodes with no edges.
+
+>>> remove_unreachable_nodes(gg)
+>>> gg.nodes()
+['Felidae', 'Lion', 'Tiger']
+
+"""
+
 from gensim.summarization.graph import Graph
 
 
 def build_graph(sequence):
+    """Creates and returns graph with given sequence of values.
+
+    Parameters
+    ----------
+    sequence : list
+        Sequence of values.
+
+    Returns
+    -------
+    Graph
```

Comment:
Concrete link to type like 
```
:class:`~gensim. ... ...
```
here and everywhere (for "gensim-defined" types).